src/apps/bridge/sensor_drivers/aanderaaSensor.cpp

```diff
@@ -52,9 +53,13 @@ void AanderaaSensor::aanderaSubCallback(uint64_t node_id, const char *topic, uin
         uint64_t sensor_reading_time_sec = d.header.sensor_reading_time_ms / 1000U;
         uint32_t sensor_reading_time_millis = d.header.sensor_reading_time_ms % 1000U;
 
+        int8_t node_position = topology_sampler_get_node_position(node_id, 1000);
+
         size_t log_buflen =
             snprintf(log_buf, SENSOR_LOG_BUF_SIZE,
                      "%" PRIx64 ","   // Node Id
+                     "%" PRIi8 ","    // node_position
+                     "aanderaa,"      // node_app_name
```

Comment:
The node's app name could be "aanderaa" or "aanderaa-dbg", right? Is there a good reason to throw away that distinguishing info? I don't have a strong opinion about it, but I'm curious whether you've already thought about that, and if so, what your reasons were for dropping any potential suffix.

Reply:
I actually have an open question for @evanShap on the spec sheet for this, but for now I  just put "aanderaa" and "soft". The reason is this snprintf is done on the bridge and the bridges sensor drivers don't know the actual app name. So it would require some extra bit of complexity to get the full name-dbg@sha.

Reply:
Thanks. Makes perfect sense to me. I look forward to hearing from Evan, and I agree with you — I'd prefer not to add more sysinfo parsing and retrieval code just to get the app name.